src/tests/LUMISPECCAL_test/TwoElectronsTopCAL.hepmc

```diff
@@ -0,0 +1,1203 @@
+HepMC::Version 3.02.05
```

Comment:
Please provide the original script instead, you should be able to run python/hepmc3 on the CI. Doing it in the same step is okay for now.

Reply:
I have a clunky cpp file to generate these hepmc events...
I wouldn't upload that.  Do you have a sample python script that I use as a template?

Reply:
```
import sys

import numpy as np
from pyHepMC3 import HepMC3

seed = 42
rng = np.random.default_rng(seed)

skip = int(sys.argv[1]) * 1000

p = 1 # GeV

writer = HepMC3.WriterAscii("test.hepmc")

for ix in range(skip + 1000):
    costheta = -1 + 2 * rng.random()
    phi = 2 * np.pi * rng.random()
    if ix < skip:
        continue

    pt = np.sqrt(1 - costheta**2) * p
    px = pt * np.cos(phi)
    py = pt * np.sin(phi)
    pz = costheta * p
    mass = 0.511e-3
    E = np.sqrt(mass**2 + p**2)
    particle = HepMC3.GenParticle(HepMC3.FourVector(px, py, pz, E))
    particle.set_pdg_id(0)
    particle.set_status(1)

    particle_in1 = HepMC3.GenParticle(HepMC3.FourVector(0, 0, 10, 10))
    particle_in1.set_pdg_id(11)
    particle_in1.set_status(4)
    particle_in2 = HepMC3.GenParticle(HepMC3.FourVector(0, 0, -100, 100))
    particle_in2.set_pdg_id(2212)
    particle_in2.set_status(4)

    vertex = HepMC3.GenVertex(HepMC3.FourVector(0., 0., 0., 0.))
    vertex.add_particle_in(particle_in1)
    vertex.add_particle_in(particle_in2)
    vertex.add_particle_out(particle)

    event = HepMC3.GenEvent(HepMC3.Units.GEV, HepMC3.Units.MM)
    event.add_vertex(vertex)
    event.set_beam_particles(particle_in1, particle_in2)

    writer.write_event(event)
```